packages/notebook/src/actions.tsx

```diff
@@ -843,7 +843,7 @@ export namespace NotebookActions {
    * This will skip any collapsed cells.
    * The existing selection will be cleared.
    */
-  export function selectAbove(notebook: Notebook): void {
+  export async function selectAbove(notebook: Notebook): Promise<void> {
```

Comment:
This (and `selecteBelow`) is a braking change in public API. Is it really needed, or can we keep it sync? It would be fine if we were targeting 5.0.

Reply:
If we keep it sync but return a `Promise<void>` instead of nothing it's easy to fix, but it also change the public API (even if it is without consequences).

It's about the tests on these actions, but it can probably be fixed in the tests themselves.